crates/egui/src/frame_state.rs

```diff
@@ -40,6 +40,23 @@ pub struct PerLayerState {
     pub widget_with_tooltip: Option<Id>,
 }
 
+#[derive(Clone, Debug)]
+pub struct ScrollTarget {
```

Comment:
I just noticed that there are now two different ScrollTarget structs in egui, scroll_area::ScrollTarget and frame_state::ScrollTarget, both have somewhat different uses. Maybe this is a bit confusing? Should I maybe rename the scroll_area one to scroll_area::ScrollingToTarget (since it tracks the currently active scroll animation)? Do you have a better naming idea @emilk?

Reply:
Yeah, let's avoid nameclashes pelase!